.buildkite/pipeline.yml

```diff
@@ -13,39 +13,77 @@ steps:
   - name: ":hammer: :linux: amd64"
     key: test-linux-amd64
     command: ".buildkite/steps/tests.sh"
+    env:
+      TEST_ANALYTICS_TOKEN_ENV_KEY: "TA_SUITE_AGENT_AMD64"
     artifact_paths: junit-*.xml
     plugins:
       docker-compose#v3.0.0:
         config: .buildkite/docker-compose.yml
         run: agent
+        env:
+          - TEST_ANALYTICS_TOKEN_ENV_KEY
```

Comment:
The use of `TOKEN` and `KEY` in here got me real confused about what this was at first. Is the funky name to get around not being able to use `TEST_ANALYTICS_TOKEN` because it's set in tests.sh? Or is there something more going on?